test/models/ui/StackingSpec.scala

```diff
@@ -23,62 +23,63 @@ import models.ocelot.stanzas._
 class StackingSpec extends BaseSpec {
 
   trait Test {
-    def stackedCo(typ: CalloutType, stack: Boolean) = Callout(typ, Phrase(), Seq.empty, stack)
+    def stackedTitle(stack: Boolean) = TitleCallout(Phrase(), Seq.empty, stack)
```

Comment:
Can you add the type on these two methods to shut up Intellij please?

Reply:
done